spec/meilisearch/index/settings_spec.rb

```diff
@@ -99,6 +103,7 @@
       expect(settings['distinctAttribute']).to be_nil
       expect(settings['stopWords']).to be_empty
       expect(settings['synonyms']).to be_empty
+      expect(settings['proximityPrecision']).to eq(default_proximityPrecision)
```

Comment:
Old version of default_proximity_precision is causing the tests to fail

Reply:
I don't know how to rerun the pipeline in GitHub

Reply:
I can rerun it but the problem is not fixed. Are you sure you pushed your local fixes?

Reply:
Mea culpa :see_no_evil: